src/openai_cli/client.py

```diff
@@ -7,11 +7,14 @@ class CompletionClient:
     TEMPERATURE = 0.1
     TIMEOUT = 60
 
-    def __init__(self, token: str, session: requests.Session) -> None:
+    def __init__(self, token: str, model:str, session: requests.Session) -> None:
         self._headers = {"Authorization": f"Bearer {token}"}
         self._session = session
+        self.model = model
 
     def generate_response(self, prompt: str, model: str = "text-davinci-003") -> str:
```

Comment:
Let's drop the model parameter here too, we don't need to have two ways to pass it.